core/nandtool/nandtool.c

```diff
@@ -159,28 +182,37 @@ static void nandtool_help(const char *prog)
 	printf("Usage: %s [options] <device>\n", prog);
 	printf("\t-e <start[:size]> - erase size block(s) starting from start\n");
 	printf("\t                    (skip size to erase one block or pass size=0 to erase whole device)\n");
+	printf("\t-j                - write jffs2 cleanmarkers (valid only with erase operation)\n");
+	printf("\t-c                - check device for bad blocks and print summary\n");
+	printf("\n");
```

Comment:
I don't think we want additional `\n` here, do we?

Reply:
we do, as the below params work only with writing (properly, should be another subsection of options)